flink-dist/pom.xml

```diff
@@ -436,6 +436,37 @@ under the License.
 					</gitDescribe>
 				</configuration>
 			</plugin>
+
+			<!-- create a symbolic link to the build target in the root directory -->
+			<plugin>
+				<groupId>com.pyx4j</groupId>
+				<artifactId>maven-junction-plugin</artifactId>
+				<version>1.0.3</version>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>link</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>unlink</id>
+						<phase>clean</phase>
+						<goals>
+							<goal>unlink</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<links>
+						<link>
+							<dst>${basedir}/../build-target</dst>
```

Comment:
I couldn't find a document stating that $basedir is deprecated, but I think you are right in the sense that the project prefix is used for everything related to the POM of the project (I think in previous versions the (now deprecated) prefix was `pom` and both the `version` and `basedir` properties are "built-ins").

We use $basedir in other places as well.